views/lotsa.erb

```diff
@@ -0,0 +1,22 @@
+<!-- This form collects three words and stores them in the morewords array -->
+<form action="" method="post" accept-charset="utf-8">
```

Comment:
For future work and exploration - this would be a great place to allow your user to specify the number of words that they want to score and you could dynamically allow them to input that number of words.